weblab/templates/includes/navbar.html

```diff
@@ -2,7 +2,11 @@
 <nav>
   <ul class="navbar">
     <li><a href="{% url 'home' %}">Home</a></li>
-    <li><a href="{% url 'experiments:list' %}">Experiments</a></li>
+    {% if user.is_authenticated %}
```

Comment:
What's the rationale for this one?

Reply:
If the user is logged in, the link takes them to a view of primarily their experiments (+moderated). If not logged in, they just get moderated entities by default.